README.md

```diff
@@ -19,7 +19,7 @@ Ushahidi 3
  [![Sponsors on Open Collective](https://opencollective.com/platform/sponsors/badge.svg)](#sponsors) 
 
 
-[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
+[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy?template=https://github.com/ushahidi/platform/tree/master)
```

Comment:
@tuxpiper should this point to master or develop? I think master but just checking in case you disagree 
do you see any issues with this approach (aside from the fact that we need to "hardcode" the branch in)

Reply:
Thanks
@lexoyo for submitting this 💯 
Question: do you think we should explicitly call out all the ENV params in the app.json ?

https://stackoverflow.com/questions/44026881/deploy-to-heroku-button-with-env-variables 
https://devcenter.heroku.com/articles/heroku-button#creating-the-app-json-file
was just checking this out and we could define the vars from https://github.com/ushahidi/platform/blob/master/.env.example in app.json too 

cc @tuxpiper

Reply:
Usually it is the master branch because the use case is "As a developer I want to host an instance on heroku to show my boss that it is easy to self host this tool"

Reply:
> we could define the vars from https://github.com/ushahidi/platform/blob/master/.env.example in app.json too

Yes sure

Reply:
maybe for that we should merge this and start an issue to discuss what we want in a second step?

my PR is just a fix, the app.json was already there